server/queue/queue-activemq/src/test/java/org/apache/james/queue/activemq/ActiveMQMailQueueBlobTest.java

```diff
@@ -95,6 +94,11 @@ public ManageableMailQueue getManageableMailQueue() {
         return mailQueue;
     }
 
+    @Override
+    public String queueName() {
```

Comment:
We should find a way to not need this IMO - I bet there is only one queue in the given recording metric, right? maybe we could expose a copy of its internal map?

Reply:
add more method for get metric with prefixName in `RecordingMetricFactory`